Poll.elm

```diff
@@ -43,7 +43,12 @@ update msg model =
     case msg of
         ChangeQuestion newQuestion ->
             if newQuestion == "is" then
-                { model | question = newQuestion, answers = addYesAndNo model.answers }
+                if not (List.member "" model.answers) then
```

Comment:
shouldn't we see if all fields are blank, and only in this case add the yes and no?